### DISCUSSION TITLE
Add user avatar to git config

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#107515] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

<img width="379" alt="Screenshot 2024-02-11 alle 11 45 24" src="https://github.com/community/community/assets/20476002/0deb7584-b281-4729-a0d3-16bb84002ea7">

It would be nice, in addition to adding name, email and commit, you could add avatars to the user.
I'm talking about the avatar where you see the github icon next to the words `Automated Publisher`.

I have the following setup on github which is run by a github action:

```sh
remote_repo="https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
git config http.sslVerify "${SSL_VERIFY}"
git config user.name "${NAME}"
git config user.email "${EMAIL}"
git remote add publisher "${remote_repo}"
git show-ref # Useful for debugging
git branch --verbose

# Install LFS hooks
git lfs install

# Publish any new files
git checkout ${BRANCH_NAME}
git add -A ${ONLY_TRACKED}
git commit -m "${COMMIT_MESSAGE}" ${NO_VERIFY}
git pull --rebase publisher ${BRANCH_NAME}
git push publisher ${BRANCH_NAME}
```

Solution1:

```
curl -s www.google.com/favicon.ico > favicon.ico
base64 favicon.ico > favicon_base64.txt
AVATAR_BASE64=$(cat favicon_base64.txt)

...
git config user.avatar "data:image/png;base64,${AVATAR_BASE64}"
...
```

Solution2:

```
curl -s "www.google.com/favicon.ico" --output favicon.ico

if [ $? -ne 0 ]; then
    echo "Error: Failed to download the favicon."
    exit 1
fi

AVATAR_BASE64=$(base64 favicon.ico)

...
git config user.avatar "data:image/png;base64,${AVATAR_BASE64}"
...
```

Solution3:

```
...
git config user.avatar "https://www.site.com/nameIcon.ext"
...
```

What do you think?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Angelk90 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by Angelk90** [reply]
@queenofcorgis : I hope someone replies so we can discuss this.
